internal/group_model.go

```diff
@@ -9,19 +9,29 @@ import (
 
 // componentGroupResourceModel maps the resource schema data.
 type componentGroupResourceModel struct {
-	ID        types.String `tfsdk:"id"`
-	GroupID   types.Int64  `tfsdk:"group_id"`
-	SpaceID   types.Int64  `tfsdk:"space_id"`
-	CreatedAt types.String `tfsdk:"created_at"`
-	UpdatedAt types.String `tfsdk:"updated_at"`
-	UUID      types.String `tfsdk:"uuid"`
-	Name      types.String `tfsdk:"name"`
+	ID      types.String `tfsdk:"id"`
+	GroupID types.Int64  `tfsdk:"group_id"`
+	SpaceID types.Int64  `tfsdk:"space_id"`
+	UUID    types.String `tfsdk:"uuid"`
+	Name    types.String `tfsdk:"name"`
```

Comment:
Not sure if we want to remove CreatedAt/UpdatedAt ? Or do these really not exist ?